### DISCUSSION TITLE
Error Popup While Working

### DISCUSSION THREAD

**Discussion by LovingHusbandW** [sumatrapdfreader/sumatrapdf#4164] [Q&A]
Hello! I am very new to this so sorry if I am posting this in the wrong place, but I am using SumatraPDF for a little project of mine in Python, and I have encountered an "issue". When trying this
```
subprocess.call(fr'{self.filePath} -print-to brother -duplex long-edge "C:\Users\wcate\OneDrive\Documents\VS Code Projects\Python Projects\PrinterMonitor\print{self.count}.pdf"')
```
The file seems to print successfully, however I get a popup from Sumatra saying that it could not find the file .\-duplex and .\long-edge for printing. I'm not 100% sure why that pops up and yet it prints fine. And to note it does not print until I clear the error pop up Doing this yeilds no popup
```
subprocess.call(fr'{self.filePath} -print-to brother "C:\Users\wcate\OneDrive\Documents\VS Code Projects\Python Projects\PrinterMonitor\print{self.count}.pdf"')
```
I would appreciate any advice! I could be totally missing out on something and just be formatting it incorrectly. And I am using the subprocess library from python to execute the command. I should also add that I want it to print duplex and on long edge and it does so yet gives me an error.

**Comment by GitHubRulesOK** [comment]
your call is wrong hence the pop-up
`-print-to brother -duplex long-edge "C:\Users\wcate\OneDrive\Documents\VS Code Projects\Python Projects\PrinterMonitor\print{self.count}.pdf"'`

in CMD should in the middle be `-print-to brother -print-settings "duplexlong" "c:\...path....filename.pdf"`
